modules/ROOT/pages/deploy-maven-3.x.adoc

```diff
@@ -81,28 +88,29 @@ This example uses placeholder values. Configure each value with values specific
 
 == Redeploy to Runtime Fabric
 
-To redeploy the application, run the same command as you did to deploy. Runtime Fabric rewrites the application you had deployed.
+To redeploy the application, run the same command as you did to deploy. Runtime Fabric rewrites the 
+application you previously deployed.
 
 == Runtime Fabric Deployment Reference
 
 [%header%autowidth.spread,cols="a,a"]
 |===
 |Parameter | Description
-|runtimeFabricDeployment | specifies a top-level element.
-| uri | specifies your Anypoint Platform URI. +
+|runtimeFabricDeployment | Specifies a top-level element.
+| uri | Specifies your Anypoint Platform URI. +
 If not set, the default value is `+https://anypoint.mulesoft.com+`.
-| muleVersion | specifies the Mule runtime version installed in your Runtime Fabric instance. +
+| muleVersion | Specifies the Mule runtime version installed in your Runtime Fabric instance. +
 Ensure that this value is at equal or higher than the minimum required Mule runtime version of your application.
-| username | specifies your Anypoint Platform username.
-| password | specifies your Anypoint Platform password.
-| applicationName | specifies the name of your application deployed in Exchange. +
+| username | Specifies your Anypoint Platform username.
+| password | Specifies your Anypoint Platform password.
+| applicationName | Specifies the name of your application deployed in Exchange. +
 This name is part of the domain of your deployed app.
-| businessGroup (optional) | specifies the business group where you want to deploy the application. If not defined, name of the master organization is used as the default.
+| businessGroup (optional) | Specifies the business group where you want to deploy the application. If not defined, the name of the master organization is used as the default.
 | skip | (boolean value) When set to true, skips the plugin deployment goal. The default value is `false`.
-| target | specifies the Runtime Fabric target name where the applicateion is deployed.
-| provider | specifies the rovider type. The only supported value for Runtime Fabric is `MC`.
-| environment | specifies the Anypoint Platform environment where you want to deploy your application.
-| properties (optional) | specifies a top-level element. +
+| target | Specifies the Runtime Fabric target name where the applicateion is deployed.
+| provider | Specifies the rovider type. The only supported value for Runtime Fabric is `MC`.
```

Comment:
typo on provider

Reply:
The wording here is weird. We could say, "Specifies the provider type; for Runtime Fabric, set this value to 'MC'".

Reply:
Got 'em. Thank you!